master/pkg/model/experiment_config.go

```diff
@@ -80,18 +81,24 @@ func (d *DeviceConfig) UnmarshalJSON(data []byte) error {
 	return errors.Wrap(json.Unmarshal(data, DefaultParser(d)), "failed to parse device")
 }
 
+// ResourcePoolName includes the (optional) ResourceManager name & ResourcePool name.
+type ResourcePoolName struct {
+	ResourceManager null.String
```

Comment:
How do we want the config to look?

The way it looks now seems a little awkward?
```yaml
resources:
    resource_pool:
        resource_manager: "name"
        resource_pool: "name"
```

Maybe just
```yaml
resources:
    resource_pool: pool
    resource_manager: name
```

Reply:
also some bikeshedding on ```resource_manager``` vs ```resource_manager_name```?

I guess it isn't ```resource_pool_name``` so just ```resource_manager``` seems closer?